components/microsoft_onedrive/sources/new-file-of-types-in-folder/mime-types.js

```diff
@@ -0,0 +1,304 @@
+// A list of the most common MIME types defined in the IANA registry. The list
```

Comment:
Consider using [mime-db](https://www.npmjs.com/package/mime-db) instead (h/t @michelle0927 for pointing this package out to me)